### PR TITLE
Create a channel stantard

### PR DESCRIPTION
**Pull request by RaulTrombin**
- [x] Needs [PR-13](https://github.com/bluerobotics/navigator-rs/pull/13)
- [x] Rebase

Like we did on navigator-lib, channels should be called with Ch*,

As here:
```
PwmChannel::Ch1
AdcChannel::Ch1
```
*As pointed on Raom meeting, DiffChannels wasn't added to the navigator AdcChannel

Also add more documentation files for set_pwm_channel_value related functions.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -247,41 +307,74 @@ impl Navigator {
...
+    /// let channels: [PwmChannel; 3] = [PwmChannel::Ch0, PwmChannel::Ch1, PwmChannel::Ch2];
+    /// let values: [u16; 3] = [200, 1000, 300];
+    /// nav.set_pwm_channels_values(channels, values); // sets the duty cycle according to the list
+    ///
+    /// ```
     pub fn set_pwm_channels_values<const N: usize>(
         &mut self,
-        channels: &[pwm_Channel; N],
-        values: &[u16; N],
+        channels: [PwmChannel; N],
```
Comment (patrickelectric) [inline src/lib.rs:373]: why we moved from reference to own the list ?

File: src/lib.rs [hunk 2]
```diff
@@ -209,7 +209,7 @@ impl Navigator {
             .expect("Error: Failed to build BMP280 device");
         bmp.zero().unwrap();
 
-        let mut spi = Spidev::open("/dev/spidev1.0").expect("Error: Failled during setting up SPI");
+        let mut spi = Spidev::open("/dev/spidev1.0").expect("Error: Failed during setting up SPI");
```
Comment (joaoantoniocardoso) [inline src/lib.rs:212]: Wrong commit?

File: src/lib.rs [hunk 3]
```diff
@@ -309,11 +309,13 @@ impl Navigator {
...
-    /// nav.pwm_enable();
-    /// nav.set_pwm_freq_prescale(100); // sets the pwm frequency to 60 Hz
-    /// nav.set_pwm_channel_value(PwmChannel::Ch0, 2048); // sets the duty cycle to 50%
+    ///let mut nav = Navigator::new();
+    ///
+    ///nav.init();
+    ///nav.pwm_enable();
+    ///
+    ///nav.set_pwm_freq_prescale(99); // sets the pwm frequency to 60 Hz
+    ///nav.set_pwm_channel_value(PwmChannel::Ch0, 2048); // sets the duty cycle to 50%
```
Comment (joaoantoniocardoso) [inline src/lib.rs:318]: It should have a space before the first character of each line of the doc comment

File: examples/raspberry-pi.rs [hunk 4]
```diff
@@ -1,4 +1,4 @@
-use navigator_rs::{pwm_Channel, Navigator, SensorData};
+use navigator_rs::{Navigator, PwmChannel};
```
Comment (joaoantoniocardoso) [inline examples/raspberry-pi.rs:1]: Pretty sure the commit message should be `examples: Update PwmChannel and remove unneeded module`

File: src/lib.rs [hunk 5]
```diff
@@ -325,6 +325,25 @@ impl Navigator {
         self.pwm.set_channel_off(channel.into(), value).unwrap();
     }
 
+    /// Like [`set_pwm_channel_value`](struct.Navigator.html#method.set_pwm_channel_value). This function
+    /// sets the Duty Cycle on a list of multiple channels.
```
Comment (joaoantoniocardoso) [inline src/lib.rs:331]: Maybe change the "on" to "for", meaning that the same value is applied to every channel of that list.

File: src/lib.rs [hunk 6]
```diff
@@ -310,6 +310,7 @@ impl Navigator {
     /// use navigator_rs::{Navigator, PwmChannel};
     ///
     /// let mut nav = Navigator::new();
+    ///
```
Comment (patrickelectric) [inline src/lib.rs:313]: Do not create images based on PRs, create images based on what they are doing.

Comment (patrickelectric) [reply src/lib.rs:313]: Besides that this is not fixing a PR, is fixing a commit in this PR.

**Review by joaoantoniocardoso — APPROVED** [review]
LGTM

**Review by patrickelectric — APPROVED** [review]

**Pull request merged**